test/extended/util/client.go

```diff
@@ -931,7 +932,15 @@ func (c *CLI) start(stdOutBuff, stdErrBuff *bytes.Buffer) (*exec.Cmd, error) {
 	}
 	cmd := exec.Command(c.execPath, c.finalArgs...)
 	cmd.Stdin = c.stdin
-	framework.Logf("Running '%s %s'", c.execPath, strings.Join(c.finalArgs, " "))
+	// The below log line prints the bearer token in the logs hence redacting
+	// the log if it contains bearer token
+	args := strings.Join(c.finalArgs, " ")
+	if strings.Contains(args, "Bearer") {
```

Comment:
Make this more specific, e.g.
```suggestion
	if strings.Contains(args, "Authorization: Bearer") {
```